src/main/java/org/beanpod/switchboard/dao/ChannelDaoImpl.java

```diff
@@ -28,6 +29,13 @@
   private final OutputChannelRepository outputChannelRepository;
   private final OutputChannelMapper outputChannelMapper;
 
+  // ChannelEntity -----------------------------------------------------------------------------
```

Comment:
I kept this minimal comment because it helps with readability. As you suggested in your last review, it may be worthwhile to refactor. I agree with you that the class itself has the smell of a god class, but this is not new and probably should have been caught in an earlier review. As much as I would love to perfect everything, the area of responsibility of this PR was becoming too big.